src/app/cli/src/coda.ml

```diff
@@ -245,13 +247,21 @@ let daemon logger =
            exit 10 )
          else Deferred.unit
        in
-       let%bind ip =
-         match ip with None -> Find_ip.find () | Some ip -> return ip
+       let%bind external_ip =
```

Comment:
can we use different names for the string and `Inet_addr.t` representations, ditto for `bind_ip`?